addons/snailmail/static/src/messaging/component/snailmail_error_dialog/snailmail_error_dialog.js

```diff
@@ -0,0 +1,106 @@
+odoo.define('snailmail.messaging.component.SnailmailErrorDialog', function (require) {
+'use strict';
+
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const Dialog = require('web.OwlDialog');
+
+const { Component } = owl;
+const { useRef } = owl.hooks;
+
+class SnailmailErrorDialog extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const message = this.env.entities.Message.get(props.messageLocalId);
+            return {
+                message: message.__state,
+                notification: message.notifications[0].__state,
```

Comment:
guard, `message` and `message.notifications[0]` may be `undefined`

Reply:
ok